### PR TITLE
Document model support

### PR DESCRIPTION
**Pull request by havatv**
Documentation for #74.

### CODE REVIEW

File: _posts/2016-07-22-repository-structure.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ order: 3
 
 In order for the plugin to be able to read the repository correctly, the 
 repository must have a specific structure. Generally the structure looks like
- this:
+this (models and R scripts are missing, but are at the same level as *image*,
```
Comment (andreasplesch) [inline _posts/2016-07-22-repository-structure.md]: consider: "The structure follows the diagram below. Note that in the diagram the *models* and *rscripts* folders are missing. They are at the same level as *image*, *processing*, *svg* and *symbol*."

File: _posts/2016-07-22-repository-structure.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ order: 3
 
 In order for the plugin to be able to read the repository correctly, the 
 repository must have a specific structure. Generally the structure looks like
```
Comment (havatv) [inline _posts/2016-07-22-repository-structure.md]:
```suggestion
repository must have a specific structure.
The general structure is shown in the illustration below.
Note that in the diagram, the ``models`` and ``rscripts`` folders are missing.
They are at the same level as ``image``, ``processing``, ``svg`` and ``symbol``.
```

File: _posts/2016-07-22-repository-structure.md (hunk 1 above)
Comment (havatv) [inline _posts/2016-07-22-repository-structure.md]:
```suggestion
```

File: _posts/2016-07-22-repository-structure.md [hunk 3]
```diff
@@ -8,7 +8,8 @@ order: 3
...
- this:
+this (models and R scripts are missing, but are at the same level as *image*,
+*processing*, *svg* and *symbol*):
```
Comment (havatv) [inline _posts/2016-07-22-repository-structure.md]:
```suggestion
```

**Pull request merged**